modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java

```diff
@@ -84,22 +98,28 @@
     private KafkaProducer<Integer, byte[]> producer;
 
     /** Handle only primary entry flag. */
-    private final boolean onlyPrimary;
+    private boolean onlyPrimary = DFLT_IS_ONLY_PRIMARY;
 
     /** Topic name. */
-    private final String topic;
+    private String topic = DFLT_TOPIC;
```

Comment:
Let's make default topic `null.

Reply:
Done. I have done the symmetric changes for KafkaToIgniteCdcStreamer too.